README.md

```diff
@@ -1 +1,95 @@
-# alerting
\ No newline at end of file
+# Alerting
+
+## Introduction
+With Hypertrace we are moving in a direction where we promote it more like an Observability platform than just a distributed tracing tool similar to Jaeger or Zipkin. So when we talk about observability, alerting is one of the main components of it. As of now we provide support for the following 
+tasks. More features would be added as we progress along in time.
+
+- Get an alert whenever there is a sudden spike/ sharp increase in latency of any operation/ API calls from or to my service.
+- Get an alert whenever there is a sudden spike in traffic/ increase in call rate to my service/ API. 
+- Get an alert whenever there is a sudden spike in errors/ increase in error rate for my service. 
+
+## Design
+Currently, we support setting up the rules via a config or via deployment. However, in the UI, we will have a way to list the alert rules and notification configuration. If we closely look at the alerting service, we see that it is simply an anomaly condition from a baseline for a particular metric. So at a high level we need a **metric anomaly definition** (alert rule) which consists of metric selection and baseline calculation. Once we have designed our alert rule, we would need a **anomaly evaluation engine** over a metric (alert rule evaluation engine) and an **alert notification engine** (notification sender) to send the notification.
```

Comment:
> So at a high level we need a

this is referring to component which already exists, please appropriately fix the language used